<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IonGunControl</class>
 <widget class="QGroupBox" name="IonGunControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>467</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica Neue</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="title">
   <string>Ion Gun Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="Source_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="emission_current_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current (0-30mA)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="emission_current_target_doubleSpinBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="emission_current_readout_doubleSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="grid_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Grid (120-200 V)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="grid_target_doubleSpinBox">
        <property name="minimum">
         <double>120.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="beam_voltage_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Voltage (0-5000 V)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="beam_voltage_target_doubleSpinBox">
        <property name="maximum">
         <double>5000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="beam_voltage_readout_doubleSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="Column_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Column</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="objective_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Objective (0-110%)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="condenser_readout_doubleSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="float_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Float (0-500 V)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="float_target_doubleSpinBox">
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="float_readout_doubleSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="objective_readout_doubleSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="objective_target_doubleSpinBox">
        <property name="maximum">
         <double>110.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="condenser_target_doubleSpinBox">
        <property name="maximum">
         <double>110.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="condenser_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Condenser (0-110%)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QGroupBox" name="Extractor_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Extractor Pressure (0-50 mPa)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="Extractor_readout_doubleSpinBox">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="Raster_mode_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Raster Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="Raster_comboBox">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Internal Raster</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>External Raster</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QGroupBox" name="Offset_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Offset (mm)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="X_off_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="X_offset_spinbox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Y_off_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Y_offset_spinbox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QGroupBox" name="Raster_size_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Raster Size (mm)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="X_Raster_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="X_raster_spinbox">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Y_Raster_label">
        <property name="font">
         <font>
          <family>Helvetica Neue</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Y_raster_spinbox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QGroupBox" name="Mode_gun_off_box">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Gun Mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QComboBox" name="Gun_mode_comboBox">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blanking</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Standby</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Active</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
